### PR TITLE
Added gold theme

### PR DESCRIPTION
**Pull request by NitinNair89**
@matteobruni I have added a gold color theme. I am good at UI but I was unable to test the colors as I could not find how to setup the entire project in my local system.

And I tried with gitpod.io but for some reason the changes did not reflect when I selected existing themes as "orange", "purple". The UI showed everything in white.

Please review and let me know if there's any more changes I have to do. Also, if you could guide me setup and view my local changes that would be nice. :slightly_smiling_face: 

This PR is for #36

### CODE REVIEW

**Comment by matteobruni** [comment]
Hi @NitinNair89,

I just fixed those themes, they were missing in the `editor.scss` file.

To test the project just `yarn && yarn start` the first time and `yarn start` every time you need to test it.

If you make changes to the editor and not the test app you can just run `yarn build` in another terminal and refresh the page.

The demo app runs on http://localhost:3030

**Review by matteobruni — COMMENTED** [review]
--

**Review by matteobruni — CHANGES_REQUESTED** [review]
The theme is missing from `editor.scss`

**Comment by NitinNair89** [comment]
Thank you so much @matteobruni :smiley: 

Let me fix the issues for my PR, will let you know.

Can you prepare one CONTRIBUTING file so beginners like me can understand how/where to make changes for theme contributions? :slightly_smiling_face:

**Comment by matteobruni** [comment]
Yes indeed, I'll work on that soon

**Comment by NitinNair89** [comment]
> The theme is missing from `editor.scss`

I added entry for gold theme. Please review.

**Comment by NitinNair89** [comment]
I am sorry. My latest commit went with my office account. Anyway to discard it? @matteobruni

**Comment by matteobruni** [comment]
I'll squash merge it so it won't appear in history, once a commit is pushed there's no way to revert it as far as I know

**Comment by NitinNair89** [comment]
> I'll squash merge it so it won't appear in history, once a commit is pushed there's no way to revert it as far as I know

Thank you @matteobruni 

I will try to contribute more themes, this looks interesting to me :slightly_smiling_face:

File: src/scss/editor.scss [hunk 1]
```diff
@@ -10,3 +10,4 @@
 @use "themes/moonlight";
 @use "themes/neulight";
 @use "themes/neudark";
+@use "themes/_gold";
```
Comment (matteobruni) [inline src/scss/editor.scss:13]: The `_` is not needed and add a new line at the end of file

**Review by matteobruni — CHANGES_REQUESTED** [review]

File: src/scss/editor.scss (hunk 1 above)
Comment (NitinNair89) [reply src/scss/editor.scss]: Oops, my bad. Fixed.

**Review by matteobruni — APPROVED** [review]

**Pull request merged**